docs/faq.rst

```diff
@@ -13,7 +13,7 @@ Yes, the `deduplication`_ technique used by
 Also, we have optional simple sparse file support for extract.
 
 If you use non-snapshotting backup tools like Borg to back up virtual machines,
-then these should be turned off for doing so. Backing up live VMs this way can (and will)
+then the VMs should be turned off before doing so. Backing up live VMs can (and will)
```

Comment:
@ThomasWaldmann Updated, thanks.

@enkore If you're not doing some kind of snapshotting then the backup tool will be reading blocks in different state of change, as other blocks are being changed in the VM.  This results in a VERY inconsistent image state that's backed-up.  If the backup operation was an atomic operation then it would work fine, but that's what snapshots end up substituting for.